packages/core/test/UniformBuffer.tests.ts

```diff
@@ -1,17 +1,16 @@
-import type {
-    IRenderingContext } from '@pixi/core';
 import {
-    Shader,
+    Buffer,
     createUBOElements,
+    generateProgram,
     generateUniformBufferSync,
+    getTestContext,
     getUBOData,
-    Buffer,
-    UniformGroup,
-    generateProgram,
-    getTestContext
-} from '@pixi/core';
+    Shader,
+    UniformGroup } from '@pixi/core';
+import { Matrix, Point, Rectangle } from '@pixi/math';
 
-import { Matrix, Rectangle, Point } from '@pixi/math';
+import type {
+    IRenderingContext } from '@pixi/core';
```

Comment:
While we are looking at linting imports/exports, this is something that has always bugged me

Is there a way to lint this sort of thing where clearly the import would fit on 1 line?

Reply:
https://github.com/pixijs/eslint-config/blob/main/index.js#L202

```json
"object-curly-newline": 0,
```

This one is related to [`object-curly-newline`](https://eslint.org/docs/latest/rules/object-curly-newline#importdeclaration-and-exportdeclaration). Maybe we can add this rule.